server/routes/index.js

```diff
@@ -64,7 +64,25 @@ router.get('/skills', skills.getSkills);
 router.get('/skills/:memberId', skills.getMemberSkills);
 router.post('/skills', skills.addSkills);
 
+/** Profile */
+
+router.get('/member/:memberId', member.getMember);
+router.patch('/member/fullname/:memberId', member.patchFullName);
+router.patch('/member/bio/:memberId', member.patchBio);
+router.patch('/member/username/:memberId', member.patchUserName);
```

Comment:
I think all these patch requests can be only on one route
```suggestion
router.patch('/member/:memberId', member.patchMember);
```
and in the body you can send either one or more field to be updated.